spec/02-integration/05-proxy/02-router_spec.lua

```diff
@@ -352,12 +352,14 @@ for _, strategy in helpers.each_strategy() do
         routes = insert_routes(bp, {
           {
             protocols = { "grpc", "grpcs" },
-            hosts = { "grpc1" },
+            hosts = { "grpc1", "grpc1:"..helpers.get_proxy_port(false, true),
+              "grpc1:"..helpers.get_proxy_port(true, true) },
```

Comment:
Are those new fixture values used in any new tests?

style: please add spaces around operators (`..`, `-`, etc...), and we should also align such array-like tables as per the rest of the codebase's codestyle:
```lua
hosts = {
  "..." .. "...",
  "..." .. "...",
}
```

Reply:
done